package.json

```diff
@@ -19,6 +19,7 @@
   },
   "scripts": {
     "test": "tape test/*.js | tap-bail | tap-spec",
+    "test:all": "tape test/*.js",
```

Comment:
What's the point of this one?

Reply:
Oh, that's me not wanting to tap-bail. I was also using tap-diff on output to spot differences in long strings. Can tidy this up